types/react-stripe-elements/index.d.ts

```diff
@@ -12,7 +12,7 @@
 //                 Hiroshi Ioka <https://github.com/hirochachacha>
 //                 Austin Turner <https://github.com/paustint>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 3.5
```

Comment:
This is probably not needed. I don't think you're using any 3.5 specific features.

Reply:
Thank you for the review.

Omit was introduced since 3.5.
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-5.html

That being said, we can define Omit by ourselves.
```
type Omit<T, K> = Pick<T, Exclude<keyof T, K>>;
```

Either is fine with me.  If you think keeping the version is beneficial, I'll follow the advice.

Reply:
I completely missed the `Omit`. In that case 3.5 makes sense. I think bumping up the version it's the best approach. Make note that some users might decide to lock to a specific version of the `d.ts` until they move to a newer ts version.